assets/dev/js/frontend/handlers/base-carousel.js

```diff
@@ -199,4 +269,66 @@ export default class CarouselHandlerBase extends SwiperHandlerBase {
 
 		this.swiper.update();
 	}
+
+	getPaginationBullets( array = true ) {
+		const paginationBullets = this.$element.find( this.getSettings( 'selectors' ).paginationBullet );
+
+		return array ? Array.from( paginationBullets ) : paginationBullets;
+	}
+
+	a11ySetWidgetAriaDetails() {
+		const $widget = this.$element;
+
+		$widget.attr( 'aria-roledescription', 'carousel' );
+		$widget.attr( 'aria-label', elementorFrontend.config.i18n.a11yCarouselWrapperAriaLabel );
+	}
+
+	a11ySetPaginationTabindex() {
+		this.getPaginationBullets().forEach( ( bullet ) => {
+			if ( ! bullet.classList.contains( 'swiper-pagination-bullet-active' ) ) {
```

Comment:
@hein-obox Question: can we get these swiper classes fro the swiper configs instead of hard coding it?
Try: `swiper.params.pagination.bulletActiveClass` for example.

Reply:
Updated.
I like it!

I haven't added the 'bullet?', because it is in a forEach loop. If there is no bullet, this line of code won't be triggered.